src/test/java/com/budgetControlGroup/budgetControl/BudgetControlApplicationTests.java

```diff
@@ -13,4 +17,13 @@
 	public void contextLoads() {
 	}
 
+	@Test
+	public void p(){
+		UserDao userDao = new UserDao();
+		User user = new User(125, "userName", "first", "last", "email", new Date(), new Date(), "pass");
```

Comment:
Is there a reason 125 is passed as the UserID every time we create a new user?

Reply:
I see your comment on the function that you're ignoring this value. If that's the case, could we pass null to avoid confusion?